openmp/runtime/src/include/omp_lib.h.var

```diff
@@ -491,13 +491,15 @@
 
         function omp_get_uid_from_device(device_num) bind(c)
           import
+          import :: c_ptr
```

Comment:
I think this should be also a `use, intrinsic ....` statement. With the `import`, it is assumed that `c_ptr` is declared in the host scope. @mjklemm, let me know if I am wrong here.